site/public/js/admin-gradeable-updates.js

```diff
@@ -223,6 +223,12 @@ $(document).ready(function () {
             }
             data[val.name] = $(val).val();
         };
+        if (data['depends_on'] !== null) {
+            data['depends_on_points'] == points;
```

Comment:
Is the `==` operator here a typo?

Reply:
It was in the suggested change in last conversation and the functionality still worked so I changed both. I will revert it.

Reply:
@williamjallen it was in the last suggested change and didn't break the functionality. So I changed both from != and = to !== and == . I have changed it back now.